### ISSUE TITLE
Add JSON Object to Project Created Activity

### ISSUE THREAD

**Issue by divine-comedian** [Giveth/impact-graph#1292]
When we call the project-created activity we currently get the following information: 

- projectTitle
- email
- projectLink
- firstName
- lastName

and we update our person's information in Ortto with: 

- email
- firstName
- lastName

## Question
 Is it possible to append JSON objects to a profile in Ortto? - this might allow us to keep track of all the projects owned by a single user in the platform - but it would require us to send that info every time a user creates a project.

I don't know how heavy of an operation that is but it is a nice to have. 

Say the object might include just 3 fields for every project: 

- project title
- project slug
- verified?
I don't know how heavy of an operation that is but it is a nice to have. 

Say the object might include just 3 fields for every project: 

- project title
- project slug
- verified?

## For now..

I'm having a meeting with some of the comms team tomorrow to understand better their product needs - will check back to see how necessary or useful this will be.

**Comment by divine-comedian** [comment]
We're going to close this one since the value it provides is unclear - we're waiting on a feature release from Ortto that would allow people to be affiliated with multiple organizations - this would solve our problem
